static/app/views/onboarding/targetedOnboarding/components/welcomeBackground.tsx

```diff
@@ -1,277 +0,0 @@
-        fill="#2f1d4a"
-      />
-      <path
-        d="M52 19a1 1 0 0 1 0-1l2-13a1 1 0 0 1 1 0l13 7 1 1a1 1 0 0 1-1 0l-16 6Zm3-13-2 12 14-5Zm40 15-15-4a1 1 0 0 1-1 0L91 4a1 1 0 0 1 1 0l4 17h-1Zm-14-4 14 3-4-15Zm34-1v-1l3-15h1l13 13a1 1 0 0 1 0 1l-17 2Zm4-14-3 13 15-2Z"
-        fill="#2f1d4a"
-      />
-      <path
-        d="M125 30a75 75 0 0 0-2-15 1 1 0 0 1 0-1l1 1a76 76 0 0 1 2 15h-1Zm-40 0a1 1 0 0 1-1 0 59 59 0 0 1 2-12 1 1 0 0 1 1 0v1a60 60 0 0 0-2 11ZM58 11l-2-3h1l2 3h-1z"
-        fill="#2f1d4a"
-      />
-      <ellipse cx="59.2" cy="12.8" rx="1" ry=".8" fill="#2f1d4a" />
-      <path fill="#2f1d4a" d="m88 13 1-4h2l-2 5-1-1z" />
-      <ellipse
-        cx="87.8"
-        cy="15.6"
-        rx="1.2"
-        ry="1.1"
-        transform="rotate(-8 88 16)"
-        fill="#2f1d4a"
-      />
-      <path fill="#2f1d4a" d="m122 9-2-4h1l1 4z" />
-      <ellipse
-        cx="122.9"
-        cy="11.4"
-        rx="1"
-        ry="1.2"
-        transform="rotate(-71 123 11)"
-        fill="#2f1d4a"
-      />
-    </Log>
-  </Container>
-);
-
-export default WelcomeBackground;
-
-const Illustration = styled(motion.svg)`
-  position: absolute;
-  /* Important is needed to override our root SVG rule.
-   * We disable clipping to allow some of the background
-   * to animate outside of the illustration during transitions */
-  overflow: visible !important;
-`;
-
-const Compass = styled(Illustration)`
-  left: 24px;
-  top: 32px;
-`;
-
-Compass.defaultProps = {
-  variants: {
-    initial: {
-      opacity: 0,
-      scale: 0.9,
-    },
-    animate: {
-      opacity: 1,
-      scale: 1,
-      transition: testableTransition({duration: 0.5}),
-    },
-    exit: {y: -120, opacity: 0},
-  },
-  transition: testableTransition({
-    duration: 0.9,
-  }),
-};
-
-const Log = styled(Illustration)`
-  right: -60px;
-  bottom: 0;
-`;
-
-Log.defaultProps = {
-  variants: {
-    initial: {
-      opacity: 0,
-      scale: 0.9,
-    },
-    animate: {
-      opacity: 1,
-      scale: 1,
-      transition: testableTransition({duration: 0.5}),
-    },
-    exit: {y: -200, opacity: 0},
-  },
-  transition: testableTransition({
-    duration: 1.1,
-  }),
-};
-
-const Container = styled(motion.div)`
-  pointer-events: none;
-  width: 100vw;
-  height: 80vh;
-  min-width: 200px;
-  max-width: 1200px;
-  min-height: 800px;
-  max-height: 900px;
-
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    display: none;
-  }
-`;
-Container.defaultProps = {
-  variants: {
-    animate: {},
-    exit: {},
-  },
-  transition: testableTransition({staggerChildren: 0.2}),
-};
```

Comment:
Now that you've deleted my nicely animated old background.

Can you please make the new background have some nice animations :)